rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 team leads, one of them must be in @paritytech/locks-review and the other in @paritytech/polkadot-review.
```

Comment:
I know this was covered in the discussion, but perhaps it's worth clarifying what this rule means for leads present in both `locks-review` and `polkadot-review` teams?